checksum/dpu/Makefile

```diff
@@ -49,28 +40,16 @@ directories:
 	@mkdir -p $(BUILDDIR)
 
 #Link
-$(EXEC): $(BUILDDIR)/$(BOOTSTRAP).$(OBJEXT) $(C_OBJECTS) $(ASM_OBJECTS)
+$(EXEC): $(C_OBJECTS) $(ASM_OBJECTS)
 	$(CC) -o $@ $^ $(LIB) $(DBG)
 
-#Assemble bootstrap
-$(BUILDDIR)/$(BOOTSTRAP).$(OBJEXT): $(BUILDDIR)/$(BOOTSTRAP).$(ASMEXT)
-	$(CC) -c -o $@ $(CFLAGS) $^ $(INC)
-
 #Compile
 $(ASM_OBJECTS): $(BUILDDIR)/%.$(OBJEXT) : $(SRCDIR)/%.$(ASMEXT)
```

Comment:
Nitpick: the assembler sources are handled exactly the same than C sources: can we try to unify the make goals ?

Reply:
Maybe more important (but still nothing to do with this PR), I don't see any assembler source file in this project. Do we need this rule at all ?

Reply:
Probably not, maybe it can be seen as an example,
but deleted ...